EIPS/eip-721.md

```diff
@@ -0,0 +1,349 @@
+
+How a client may determine which token and deed contracts are well-known is outside the scope of this standard.
+
+A mechanism is provided to associate deeds with URIs. We expect that many implementations will take advantage of this to provide metadata for each deed. The image size recommendation is taken from Instagram, they probably know much about image usability. The URI MAY be mutable (i.e. it changes from time to time). We considered a deed representing ownership of a real-world asset, in this case metadata about such an asset may naturally change.
+
+Metadata is returned as a string value. Currently this is only usable as calling from `web3`, not from other contracts. This is acceptable because we have not considered a use case where an on-blockchain application would query such information.
+
+*Alternatives considered: put all metadata for each deed on the blockchain (too expensive), use URL templates to query metadata parts (URL templates do not work with all URL schemes, especially P2P URLs), [multiaddr network address](https://github.com/multiformats/multiaddr) (not mature enough)*
+
+**Community consensus**
+
+A significant amount of discussion occurred on [the original ERC-721](ERC-721 issue), additionally we held a live meeting [on Gitter](https://gitter.im/ethereum/ERCs?at=5a62259b5ade18be3998eec4) that had good representation and [was](https://www.reddit.com/r/ethereum/comments/7r2ena/friday_119_live_discussion_on_erc_nonfungible/) [well](https://gitter.im/ethereum/EIPs?at=5a5f823fb48e8c3566f0a5e7) [advertised](https://github.com/ethereum/eips/issues/721#issuecomment-358369377) in relevant communities. Thank you  to the participants:
+
+- [@ImAllInNow](https://github.com/imallinnow) Rob from DEC Gaming / Presenting Michigan Ethereum Meetup Feb 7
+- [@Arachnid](https://github.com/arachnid) Nick Johnson
+- [@jadhavajay](https://github.com/jadhavajay) Ajay Jadhav from AyanWorks
+- [@superphly](https://github.com/superphly) Cody Marx Bailey - XRAM Capital / Sharing at hackathon Jan 20 / UN Future of Finance Hackathon.
+
+A second event was held at ETHDenver 2018 to discuss deed standards (notes to be published).
+
+We have been very inclusive in this process and invite anyone with questions or contributions into our discussion. However, this standard is written only to support the identified use cases which are listed herein. 
+
+## Backwards Compatibility
+
+We have adopted `balanceOf`, `totalSupply`, `name` and `symbol` semantics from the [ERC-20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) specification. An implementation may also include a function `decimals` that returns `uint8 0` if its intention is to be more compatible with ERC-20 while supporting this standard. However, we find it contrived to require all ERC-721 implementations to support the `decimals` function.
+
+Example deeds implementations as of February 2018:
```

Comment:
ENS is also one of the earliest implementations, and even calls it deeds. Of course we would support making ENS names as ERC721 compatible when we update the registrar

Reply:
@alexvandesande Thank you. This is very helpful to have you discuss here. I have read https://github.com/ethereum/EIPs/blob/master/EIPS/eip-137.md but did not see a reference to deeds, do you have another reference? Also could you please clarify your role in the ENS project.